classes.py

```diff
@@ -14,9 +14,10 @@ def __init__(self):
     self.play_area_height = 640
     self.border_colour = "#06001a"
     self.bg_colour = "#00334d"
+    self.text_colour = "#bfbfbf"
+    self.font = pygame.font.SysFont(None, 40)
```

Comment:
This is only used for the static score text since all the displays create their own fonts. `game.font` sounds like a general font for multiple game things and could be misleading. Unless you plan to use it for some other text as well?

Reply:
If I make some visual instructions and stuff I think I will use these for that